config/nrfconnect/chip-module/CMakeLists.txt

```diff
@@ -129,7 +129,7 @@ matter_add_gn_arg_bool  ("chip_enable_nfc"                        CONFIG_CHIP_NF
 matter_add_gn_arg_bool  ("chip_enable_ota_requestor"              CONFIG_CHIP_OTA_REQUESTOR)
 matter_add_gn_arg_bool  ("chip_persist_subscriptions"             CONFIG_CHIP_PERSISTENT_SUBSCRIPTIONS)
 matter_add_gn_arg_bool  ("chip_monolithic_tests"                  CONFIG_CHIP_BUILD_TESTS)
-matter_add_gn_arg_bool  ("chip_inet_config_enable_tcp_endpoint"   CONFIG_CHIP_BUILD_TESTS)
+matter_add_gn_arg_bool  ("chip_inet_config_enable_tcp_endpoint"   FALSE)
```

Comment:
This change should be explained in the PR summary: why did we change this from generally "yes for tests" to "never". Describe why this is a better path than removing the test for NRF (e.g. splitting tests into 2 and have one of the cpp files not included for nrf)

Reply:
Added some justification in the PR summary. A future approach would be to split the file up but for now, since the platform is not using TCP, it is disabled.